### ISSUE TITLE
Copy-to-clipboard does not select DataTable Headers

### ISSUE THREAD

**Issue by archiewood** [evidence-dev/evidence#1224]
## Steps To Reproduce

1. run the template project 
2. attempt to Ctrl+A, Ctrl+C, Ctrl+V the data into a google doc

### Environment
```
npm -v
9.8.1
node -v
v20.6.1
@evidence-dev/core-components@2.0.2
@evidence-dev/evidence@21.0.3
```
- OS: MacOS Ventura
- Browser: Firefox, Chrome

## Expected Behavior

Whole table is copied including headers

## Actual Behaviour

Only the body rows are copied. header rows are not copied

![CleanShot 2023-09-29 at 23 01 17](https://github.com/evidence-dev/evidence/assets/58074498/deb74100-675f-4348-be8b-4db42a2d1a5a)

**Comment by archiewood** [comment]
I suspect https://github.com/evidence-dev/evidence/blame/e22a6592cc97af0e5972a49562b6f9db1bdd8b49/packages/core-components/src/lib/unsorted/viz/DataTable.svelte#L827

**Comment by NewtonFernandis** [comment]
@archiewood I will work on this, Could you please assign this to me?

**Comment by archiewood** [comment]
It's yours!

**Comment by archiewood** [comment]
Closed by #1226
